morpheushelper/cogs/joined_info.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime
+
+from PyDrocsid.database import db_thread, db
+from PyDrocsid.translations import translations
+from discord import Member
+from discord.ext import commands
+from discord.ext.commands import Cog, Context, Bot, guild_only
+
+from models.mod import Join, Kick
+from dateutil.relativedelta import relativedelta
+
+
+def date_diff_to_str(date1: datetime, date2: datetime):
+    rd = relativedelta(date1, date2)
+    if rd.years:
+        return translations.f_joined_years(rd.years)
+    if rd.months:
+        return translations.f_joined_months(rd.months)
+    if rd.weeks:
+        return translations.f_joined_weeks(rd.weeks)
+    return translations.joined_days
+
+
+class JoinedInfoCog(Cog):
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @commands.command()
+    @guild_only()
+    async def joined(self, ctx: Context, user: Member = None):
+        """
+        Returns the date on which the user joined the server
+        """
+
+        if not user:
+            user = ctx.author
+        last_auto_kick = await db_thread(lambda: db.query(Kick).filter(Kick.mod.is_(None), Kick.member == user.id)
+                                         .order_by(Kick.timestamp.desc()).first())
```

Comment:
please use `filter_by` instead of `filter`:
```suggestion
        last_auto_kick = await db_thread(lambda: db.query(Kick, mod=None, member=user.id)
                                         .order_by(Kick.timestamp.desc()).first())
```